packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -12,9 +12,9 @@ const lerna = require('lerna');
 const build = require('@loopback/build');
 
 describe('app-generator (SLOW)', function() {
-  const generator = path.join(__dirname, '../generators/app');
-  const rootDir = path.join(__dirname, '../../..');
-  const sandbox = path.join(__dirname, '../../_sandbox');
+  const generator = path.join(__dirname, '../../generators/app');
+  const rootDir = path.join(__dirname, '../../../..');
+  const sandbox = path.join(__dirname, '../../../_sandbox');
```

Comment:
While you're here can you change `_sandbox` to `.sandbox`

Reply:
I'm guessing it's a stylistic change?

Reply:
Yep. Been meaning to change all `_sandbox` to just `.sandbox` and then we can have just `.sandbox` in the `.gitignore`.